CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
```

Comment:
please change directly with `## [v2.0.1] - 2020-07-29` and add refs also above!
Thanks!

Reply:
No! :) 
You should add new title for the new tag and leave `Unreleased`.

example:

```
## [Unreleased]	


## [v2.0.1] - 2020-07-29

### Changed

* fix last argument of generate method to be back-compatible with v1

```

and at the bottom of the page add and modify:
```
[Unreleased]: https://github.com/badges/poser/compare/v2.0.1...HEAD
[v2.0.1]: https://github.com/badges/poser/tree/v2.0.1
```